pom.xml

```diff
@@ -2420,17 +2473,6 @@
                 </rules>
               </configuration>
             </execution>
-            <execution>
-              <id>enforce-no-duplicate-dependencies</id>
-              <goals>
-                <goal>enforce</goal>
-              </goals>
-              <configuration>
-                <rules>
-                  <banDuplicatePomDependencyVersions/>
-                </rules>
-              </configuration>
-            </execution>
```

Comment:
Is there any way to provide specific exclusions for the enforcement? It's a shame to have to completely turn off the rule for this.

Reply:
I'm not sure whether exclusion rules for this. Just tried on my laptop with this and compilation for Hadoop 2.7 works, probably because there are lots of changes since I initially removed this. Let me try it once again with Spark CI.